.github/workflows/tool-test-general.yml

```diff
@@ -14,147 +14,16 @@ on:
-          key: ${{ runner.os }}-${{ steps.android-sdk-version.outputs.revision }}
-      - name: Setup Android SDK (cold cache)
-        if: env.ACT && steps.android-sdk-setup.outputs.cache-hit != 'true'
-        uses: android-actions/setup-android@9fc6c4e9069bf8d3d10b2204b1fb8f6ef7065407
-        with:
-          packages: 'tools platform-tools platforms;${{env.ANDROID_PLATFORM}} build-tools;${{env.ANDROID_BUILDTOOLS}}'
-          log-accepted-android-sdk-licenses: false
-          cmdline-tools-version: ${{ env.ANDROID_TOOLS_VERSION }}
-      - name: Setup Android SDK (warm cache)
-        if: env.ACT && steps.android-sdk-setup.outputs.cache-hit == 'true'
-        run: |
-          echo "$GITHUB_WORKSPACE/.android/sdk/cmdline-tools/${{ env.ANDROID_TOOLS_VERSION }}/bin" >> "$GITHUB_PATH"
-          echo "$GITHUB_WORKSPACE/.android/sdk/platform-tools" >> "$GITHUB_PATH"
-
-      - name: Add `flutter` to the PATH
-        run: |
-          echo "$GITHUB_WORKSPACE/flutter/bin" >> "$GITHUB_PATH"
-
-      - name: Setup PUB_CACHE environment variable
-        run: |
-          echo "PUB_CACHE=$GITHUB_WORKSPACE/.pub-cache" >> $GITHUB_ENV
-
-      # Get the Flutter revision. This is the key for the cache for artifacts
-      # under bin/cache
-      - name: Get Flutter version
-        id: flutter-revision
-        working-directory: ${{ github.workspace }}/flutter
-        run: |
-          echo "revision=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
-      - name: Flutter artifacts cache
-        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684
-        id: flutter-artifacts
-        with:
-          path: ${{ github.workspace }}/flutter/bin/cache
-          key: ${{ runner.os }}-flutter-${{ steps.flutter-revision.outputs.revision }}
-
-      - name: pub deps hash
-        id: pub-deps-hash
-        working-directory: ${{ github.workspace }}/flutter
-        run: |
-          # Generate stable hash of pubspec.yaml files
-          find dev examples packages -name "pubspec.yaml" -print0 | sort -z | xargs -0 cat | sha256sum  >> "$RUNNER_TEMP/pub_deps_sha"
-          echo "revision=$(cat "$RUNNER_TEMP/pub_deps_sha")" >> "$GITHUB_OUTPUT"
-      - name: pub package cache
-        uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684
-        id: pub-cache
-        with:
-          path: |
-            ${{ github.workspace }}/.pub-cache
-            ${{ github.workspace }}/flutter/**/.dart_tool
-            ${{ github.workspace }}/flutter/**/pubspec.lock
-          key: ${{ runner.os }}-pub-${{ steps.pub-deps-hash.outputs.revision }}
-
-      - name: Flutter Doctor
-        working-directory: ${{ github.workspace }}/flutter
-        run: |
-          flutter doctor
-
-      - name: update-packages (online)
-        if: steps.pub-cache.outputs.cache-hit != 'true'
-        working-directory: ${{ github.workspace }}/flutter
-        run: |
-          flutter update-packages
+      - uses: actions/checkout@v4
```

Comment:
Why is this still here? I would have thought that the checkout in the composite action would do this.

Reply:
Another option is publishing the action. I don't believe we want to do that.

Reply:
I think I see. You have to check out the repo in order to refer to the composite action that is defined in a file in the repo?

Reply:
lgtm with that as a comment =)

Reply:
done